paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerConfigurationHandler.kt

```diff
@@ -85,10 +84,17 @@ internal class FlowControllerConfigurationHandler @Inject constructor(
         }
 
         AnalyticsRequestFactory.regenerateSessionId()
+
+        val isDecoupling = initializationMode is PaymentSheet.InitializationMode.DeferredIntent
+        eventReporter.onInit(
+            configuration = configuration,
+            isDecoupling = isDecoupling,
+        )
```

Comment:
Trying to align this as much as possible with iOS. iOS sends this event when `FlowController` is being initialized, which we can’t do due to API differences. The closest thing we have is the `configure` method.